ThreeSum.java

```diff
@@ -0,0 +1,32 @@
+import java.util.*;
+
+public class ThreeSum {
+    //time complexity" for sorting its n(log n) // not sure of the other part
```

Comment:
Kindly write the approach that you are using first. 
For two pointers approach + sorting => TC = O(nlog n) + O(n^2) which is the optimised one and you don't require any extra space here as well because you will be moving your pointer till the time you are not finding the unique element , so you don't require to handle duplicate pairs.
for BS + sorting => TC = O(n^2 log n)